Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -1142,6 +1142,29 @@ def update_edl_command(args: dict, params: dict):
     return hr, {}, {}
 
 
+@debug_function
+def get_edl_command(args: dict, params: dict):
+    results = {}
+    with APP.test_client() as c:
+        resp = c.get('/')
+        header_dict = {}
+        for entry in resp.headers.to_wsgi_list():
+            if entry[0] and entry[1]:
+                header_dict[entry[0]] = entry[1]
+        results = {
+            "Values": (resp.text).splitlines(),
+            "ResponseHeaders": header_dict,
+            "Count": len((resp.text).splitlines())
+        }
+    return CommandResults(
+        outputs_prefix="Edl",
+        outputs_key_field="Values",
+        outputs=results,
+        raw_response=results,
+        ignore_auto_extract=True
+    )
```

Comment:
Let's return a file instead of the results to context. We may have a lot of results and this could make the command very heavy on the system. here is a short info on returning a file entry https://xsoar.pan.dev/docs/integrations/context-and-outputs#return-file